providers/github/github.go

```diff
@@ -44,6 +44,19 @@ func (p *Provider) addProductPath(product string) {
 		}
 		p.URL = p.baseURL
 		return
+	case "trending":
+		var language string
+		if language = p.Ctx.String("language"); language != "" {
+			p.join(fmt.Sprintf("trending/%s", language))
+		} else {
+			p.join(fmt.Sprintf("trending"))
+		}
+		var since string
+		if since = p.Ctx.String("since"); since != "" {
+			param := url.Values{}
+			param.Add("since", since)
+			p.URL.RawQuery = param.Encode()
+		}
```

Comment:
```golang
if since = p.Ctx.String("since"); since != "" {
			param := url.Values{}
			param.Add("since", since)
			p.URL.RawQuery = param.Encode()
}
if language = p.Ctx.String("language"); language != "" {
     p.join(fmt.Sprintf("trending/%s", language))
      return
 }
p.join(fmt.Sprintf("trending"))
 
```

Reply:
I am not sure this works or not.

Reply:
I just Resolved it!